python/ray/worker.py

```diff
@@ -858,7 +855,6 @@ def init(
         mode=driver_mode,
         log_to_driver=log_to_driver,
         worker=global_worker,
-        driver_object_store_memory=_driver_object_store_memory,
```

Comment:
this changes public API, should we leave it there and add some warning instead?

Reply:
Yea probably should log a warning that it will be removed.